eos_cli_config_gen: adding gnmi template and fixing TA config for CVaaS

noredistribution opened the pull request:
## Change Summary
Added management api gnmi template and fixed TerminAttr CVaaS config

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [x] Documentation content changes
- [ ] Other (please describe):

## Related Issue(s)
Fixes #430 
Fixes #302 


## Component(s) name

## Proposed changes
<!--- Describe your changes in detail -->

## How to test
I've used the `dc-fabric-deploy-cvp.yml` playbook from https://github.com/arista-netdevops-community/ansible-webinar-february-2020 

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] I have read the [**CONTRIBUTING**](https://www.avd.sh/docs/contributing/) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed ([`pre-commit`](https://www.avd.sh/docs/installation/development/#python-virtual-environment), `make linting` and `make sanity-lint`).
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly

carlbuchmann committed:
fix #352

carlbuchmann committed:
fix typos

carlbuchmann committed:
Update CI artifacts

carlbuchmann committed:
add management-ssh CI test

noredistribution committed:
eos_cli_config_gen: adding gnmi template and fixing TA config for CVaaS

titom73 committed:
Update docker stack for Docker In Docker support

titom73 committed:
Update Makefile

- Update to support non-root user in container.
- Update to pull image when stack is starting.

titom73 committed:
Cleanup Makefile

carlbuchmann committed:
update CI artifacts

carlbuchmann commented on ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/terminattr-cloud.cfg:
```diff
@@ -1,7 +1,7 @@
 !
 daemon TerminAttr
-   exec /usr/bin/TerminAttr -cvaddr=apiserver.corp.arista.io:9910 -cvcompression=gzip -taillogs -cvauth=token-secure,/tmp/onboardingtoken1 -smashexcludes=ale,flexCounter,hardware,kni,pulse,strata -ingestexclude=/Sysdb/cell/1/agent,/Sysdb/cell/2/agent  -cvvrf=mgt
+   exec /usr/bin/TerminAttr -cvaddr=apiserver.corp.arista.io:443 -cvcompression=gzip -taillogs -cvauth=token-secure,/tmp/cv-onboarding-token -smashexcludes=ale,flexCounter,hardware,kni,pulse,strata -ingestexclude=/Sysdb/cell/1/agent,/Sysdb/cell/2/agent  -cvvrf=mgt
```
@noredistribution please validate generated output!

noredistribution replied:
yes, that's the correct config, works on my CVaaS tenant

carlbuchmann reviewed (commented):
LGTM - please validate CI output, and we can merge!

titom73 committed:
Update documentation templates as per #428

- Remove blank line when conditions are not met
- Set default message when missing
- Fix some rendering issues

titom73 committed:
Update CI artifacts

titom73 committed:
Update template syntax

titom73 committed:
Update CI artifacts

titom73 reviewed (changes requested):
Can you please build GNMI's documentation file as well ? In a similar way of PR #443

aphillipps committed:
Fixed titles in eos_cli_config_gen documentation

onurgashi committed:
Delete redundant prefix-list that is not used in any route-map
Fixes #357

noredistribution committed:
eos_cli_config_gen: update gnmi documentation

titom73 committed:
Update documentation update

titom73 committed:
Update CI artifacts

titom73 committed:
Merge branch 'devel' into add_gnmi_fix_ta
